metrics/prometheus_fake.go

```diff
@@ -688,6 +688,11 @@ func (p testSubcontainersInfoProvider) GetRequestedContainersInfo(string, v2.Req
 								PMU:    "uncore_imc_0",
 							},
 						},
+						PerfErrors: []info.PerfError{{
```

Comment:
I know that there was an assumption that each error code exiting for the same action and event will be available only once in metrics.  Do you think that it is worth to add the one more element with the same error core, action and event to show that only unique metric will be available?